app/assets/appfilter.xml

```diff
@@ -2819,6 +2820,7 @@
   <item component="ComponentInfo{com.teacapps.barcodescanner/net.qrbot.ui.scan.ScanActivity}" drawable="qr_reader" name="QR Reader"/>
   <item component="ComponentInfo{com.tomfong.simpleqr/com.tomfong.simpleqr.MainActivity}" drawable="qr_reader" name="Simple QR"/>
   <item component="ComponentInfo{org.barcodescanner/com.example.barcodescanner.feature.tabs.BottomTabsActivity}" drawable="qr_reader" name="QR &amp; Barcode Scanner"/>
+  <item component="ComponentInfo{[com.barclays.android.barclaysmobilebanking]/[com.barclays.android.registration.NRegSplashActivity]}" drawable="barclays" name="Barclays"/>
```

Comment:
`<item component="ComponentInfo{com.barclays.android.barclaysmobilebanking/com.barclays.android.registration.NRegSplashActivity}" drawable="barclays" name="Barclays"/> `

Reply:
Thanks for catching this I have now updated to the code to not include `[ ]` 👍